### PR TITLE
RP2-476 Feature added

### PR DESCRIPTION
**Pull request by abdulrkalam**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/CaseNotesResourceController.kt [hunk 1]
```diff
@@ -66,4 +67,38 @@ class CaseNotesResourceController(
...
+        ],
+      ),
+    ],
+  )
+  suspend fun getCaseNotesCreators(
+    @PathVariable("prisonerId")
+    @Parameter(required = true)
+    prisonerId: String,
+    @PathVariable("pathway")
+    @Parameter(required = true, description = "Get Case notes Creators for a specific pathway, property supported are ACCOMMODATION, ATTITUDES_THINKING_AND_BEHAVIOUR, CHDFAMCOMCHILDREN_FAMILIES_AND_COMMUNITY, DRUGS_AND_ALCOHOL, EDUCATION_SKILLS_AND_WORK, FINANCE_AND_ID, HEALTH, GENERAL ")
```
Comment (JamesBoobierMOJ) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/CaseNotesResourceController.kt:101]: CHDFAMCOMCHILDREN_FAMILIES_AND_COMMUNITY is a typo?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/CaseNotesApiService.kt [hunk 2]
```diff
@@ -115,22 +125,32 @@ class CaseNotesApiService(
...
+    searchSubTerm: String?,
+    nomisId: String,
+    days: Int,
+  ): Flow<List<CaseNote>> = flow {
     var page = 0
     var uriValue = "/case-notes/{nomisId}?page={page}&size={size}&type={type}"
     val pattern = DateTimeFormatter.ISO_LOCAL_DATE_TIME // ofPattern(DateTimeFormatter.ISO_LOCAL_DATE_TIME.toString())
     val startDate = LocalDate.now().minusDays(days.toLong()).atStartOfDay().format(pattern)
     val endDate = LocalDate.now().plusDays(1).atStartOfDay().format(pattern)
+    log.fatal("Start DAte : $startDate")
```
Comment (JamesBoobierMOJ) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/CaseNotesApiService.kt:139]: Should be debug level logging

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/CaseNotesResourceController.kt (hunk 1 above)
Comment (abdulrkalam) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/CaseNotesResourceController.kt:101]: Done the Correction

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/CaseNotesApiService.kt (hunk 2 above)
Comment (abdulrkalam) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/CaseNotesApiService.kt:139]: Done the Correction

**Review by JamesBoobierMOJ — DISMISSED** [review]

**Review by JamesBoobierMOJ — APPROVED** [review]

**Pull request merged**